charmcraft/commands/store/__init__.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For further info, check https://github.com/canonical/charmcraft
+
+"""Commands related to the Store, a thin layer above real functionality."""
+
+import logging
+
+from charmcraft.cmdbase import BaseCommand
+from charmcraft.commands.store.client import Client
+
+logger = logging.getLogger('charmcraft.commands.store')
+
+
+class LoginCommand(BaseCommand):
+    """Log into the store."""
+    name = 'login'
+    help_msg = "login in the Ubuntu Single Sign On"
+
+    def run(self, parsed_args):
+        """Run the command.
+
+        The login happens on every request to the Store (if current credentials were not
+        enough), so here we just exercise the simplest command regarding developer identity.
+        """
+        client = Client()
+        client.clear_credentials()
```

Comment:
do people expect login to clear credentials? or is login while logged in more of "already logged in as..."

eg 'ensure I'm logged in.

Reply:
`snapcraft login` asks your credentials twice if you run it twice, so I guess this is normal for these Canonical's tools.

Reply:
IOW, it doesn't force you to logout first if you want to change your "personality".

Reply:
All that said, this is something UX will have a final word about, so I wouldn't care too much right now.